automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala

```diff
@@ -196,7 +197,7 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
     //Test to check if extensions are installed correctly
     //Using nbtranslate extension from here:
     //https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/nbTranslate
-    "should install a single user specified notebook extension" in {
+    "should install a single user specified notebook extension" taggedAs Tags.SmokeTest in {
```

Comment:
I might remove this as a smoke test -- the extension functionality is not widely used, and should be exercised anyway with the JupyterLab test.